src/encord_active/app/common/components/metric_summary.py

```diff
@@ -28,13 +30,63 @@
 )
 from encord_active.lib.metrics.utils import (
     MetricData,
+    MetricSchema,
     MetricScope,
     load_available_metrics,
 )
 
 _COLUMNS = MetricWithDistanceSchema
 
 
+def render_2d_metric_plots(metrics_data_summary: MetricsSeverity):
+    with st.expander("2D metrics view", True):
+        metric_selection_col, scatter_plot_col = st.columns([2, 5])
+        metric_names = [metric_name for metric_name in metrics_data_summary.metrics.keys()]
+        x_metric_name = metric_selection_col.selectbox("x axis", metric_names, index=0)
+        y_metric_name = metric_selection_col.selectbox("y axis", metric_names, index=1)
+        trend_selected = metric_selection_col.checkbox(
+            "Show trend",
+            value=True,
+            help="Draws a trend line to demonstrate the relationship between the two metrics.",
+        )
+
```

Comment:
We use the `MetricSchema.[whatever]` in many places here. Perhaps we should type the `MetricOutlierInfo.df` as a `MetricSchema`?

Reply:
I typed the `MetricOutlierInfo.df` as `MetricSchema`. However, I am not sure how that will relate to the usage of `MetricSchema.[whatever]` ?

Reply:
I changed the type of `MetricOutlierInfo.df` to `MetricWithDistanceSchema` as it is more appropriate.